.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
should not be there. It is related to your setup, and so belongs to your computer. See https://help.github.com/articles/ignoring-files/

Reply:
oups sorry bad place